### PR TITLE
Fix to pre-commit docs

### PR DESCRIPTION
**Pull request by potiuk**
Doc only change!

### CODE REVIEW

**Comment by potiuk** [comment]
@sprzedwojski @nuclearpinguin @mik-laj  -> easy doc change :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -118,9 +118,13 @@ We are using a number of checks for quality checks of the code. They are verifie
 also you can install pre-commit hook by running:
 
 `pre-commit install`
-`pre-commit install --type pre-push`
 
-You can run all the checks manually by running:
+For pre-commit hook and
```
Comment (sprzedwojski) [inline CONTRIBUTING.md]:
```suggestion
For pre-push hook run
```

Didn't you mean pre-push here?

Comment (potiuk) [reply CONTRIBUTING.md]: Oh yeah. Too fast.

Comment (potiuk) [reply CONTRIBUTING.md]: Fixed

**Pull request merged**